### ISSUE TITLE
missing LSB header in init-script?

### ISSUE THREAD

**Issue by tiberius** [signal18/replication-manager#158]
Hello, 

I recently installed https://github.com/tanji/replication-manager/releases/download/1.1.0-preview1/replication-manager_1.1.0-dev-202-gbec6bf5-bec6bf5_amd64.deb and noticed following error message during install to a debian jessie system:

```
insserv: warning: script 'replication-manager' missing LSB tags and overrides
insserv: There is a loop between service php-5.6.2-fpm and replication-manager if stopped
insserv:  loop involving service replication-manager at depth 2
insserv:  loop involving service php-5.6.2-fpm at depth 1
insserv: Stopping replication-manager depends on php-5.6.2-fpm and therefore on system facility `$all' which can not be true!
insserv: exiting now without changing boot order!
```

There's nothing really related to the php service, it's just missing the LSB Header: https://wiki.debian.org/LSBInitScripts

If I just add
```
### BEGIN INIT INFO
# Provides:          replication-manager
# Required-Start:    $all
# Required-Stop:     $all
# Default-Start:     2 3 4 5
# Default-Stop:      0 1 6
# Short-Description: starts replication-manager
# Description:       starts the replication-manager
### END INIT INFO
```
right after the shebang, the insserv call succeeds.

**Comment by tanji** [comment]
Hi @tiberius,

Well, Jessie uses systemd, so the systemctl script should be used instead.
However, it seems a good idea to add that in the regular init script.
Thanks for the suggestion!

**Comment by tiberius** [comment]
Hey, 

yeah, jessie by default uses systemd (we aren't, that's why I'm tripping on it). So would be nice to have it. I can try to create a PR if you want?

**Comment by tanji** [comment]
Fixed in  cfe501f
